run.py

```diff
@@ -72,6 +73,17 @@ def install_secret_keys(application, secret_session='secret_key', secret_csrf='s
 install_secret_keys(app)
 
 
+def username(u):
+    user_id = str(u)
```

Comment:
Is this specific cast required? a str.format below (which requires no casting) might be clearer on intent.